neqo-transport/src/fc.rs

```diff
@@ -363,13 +418,14 @@ impl ReceiverFlowControl<StreamId> {
                 max_data: max_allowed,
             }));
             self.frame_sent(max_allowed);
+            self.max_allowed_sent_at = Some(now);
```

Comment:
The name of this is a bit strange.  It's just the time at which you last sent a frame.  Would `last_update` make more sense?

Reply:
Renamed in 397fdd34b83b20390d7a1fd061d70d0af005b998.